### DISCUSSION TITLE
Rounded corners with line function

### DISCUSSION THREAD

**Discussion by psads-git** [cetz-package/cetz#673] [Q&A] [answered]
The following code produces the arrow depicted below. I suspect that, from function `line`, we cannot get the corner rounded (as we can get with `rect` using `radius` parameter). If I am right, would it be good idea to add that feature to function `line` as well?

```
#import "@preview/cetz:0.2.2"

#cetz.canvas({
  import cetz.draw: *

  line((0,0), (2, 0), (2,2), mark: (end: ">", fill: black))
})
```

![image](https://github.com/user-attachments/assets/11006494-c8f7-448b-bd04-7d25a71b8181)

**Comment by johannes-wolf** [comment, marked as answer]
The idea is to add this feature via modifiers to any drawable, see #467 (not before 0.4.0).

**Reply by psads-git** [reply]
Thanks, @johannes-wolf, for your reply! 

Those are great news! Meanwhile, one can use the workaround suggested by @fenjalien [here](https://github.com/cetz-package/cetz/issues/514#issuecomment-1963106507)
